tests/array_test.go

```diff
@@ -66,6 +66,46 @@ func TestSimpleArray(t *testing.T) {
 	require.NoError(t, rows.Err())
 }
 
+type customArr []customStr
+
+func TestCustomArray(t *testing.T) {
+	conn, err := GetNativeConnection(nil, nil, &clickhouse.Compression{
```

Comment:
example of usage # 3 - for column Array(Enum) we can use custom string type
It could be useful